main/src/ca/uwaterloo/flix/language/dbg/DocAstFormatter.scala

```diff
@@ -324,9 +325,9 @@ object DocAstFormatter {
       case arrow@Type.Arrow(_, _) =>
         val (curriedArgs, res) = collectArrowTypes(arrow)
         val formattedArgs = curriedArgs.map {
-          case ts@(Type.Tuple(_) :: Nil) =>
+          case ts@Type.Tuple(_) :: Nil =>
```

Comment:
So `::` has higher precedence than `@`? This confuses me.

Reply:
Scala complained that the parentheses were redundant

Reply:
Another reason not to use @

Reply:
What was the first reason?

Reply:
I dont like it :)